core/blockchain.go

```diff
@@ -2417,6 +2404,36 @@ func (bc *BlockChain) UpdateValidatorVotingPower(
 	return validatorStats, nil
 }
 
+func (bc *BlockChain) ComputeAndUpdateAPR(
+	block *types.Block, now *big.Int,
+	wrapper *staking.ValidatorWrapper, stats *staking.ValidatorStats,
+) error {
+	if aprComputed, err := apr.ComputeForValidator(
+		bc, block, wrapper,
+	); err != nil {
+		if errors.Cause(err) == apr.ErrInsufficientEpoch {
+			utils.Logger().Info().Err(err).Msg("apr could not be computed")
```

Comment:
Is it an info or error?

Reply:
info is enough as there is no stop return for this case.